doc/src/neigh_modify.rst

```diff
@@ -188,15 +196,39 @@ atom can have.
 The *binsize* option allows you to specify what size of bins will be
 used in neighbor list construction to sort and find neighboring atoms.
 By default, for :doc:`neighbor style bin <neighbor>`, LAMMPS uses bins
-that are 1/2 the size of the maximum pair cutoff.  For :doc:`neighbor style multi <neighbor>`, the bins are 1/2 the size of the minimum pair
-cutoff.  Typically these are good values for minimizing the time for
+that are 1/2 the size of the maximum pair cutoff.  For :doc:`neighbor style multi <neighbor>`, 
+the bins are 1/2 the size of the collection interaction cutoff.  
+Typically these are good values for minimizing the time for
 neighbor list construction.  This setting overrides the default.
 If you make it too big, there is little overhead due to
 looping over bins, but more atoms are checked.  If you make it too
 small, the optimal number of atoms is checked, but bin overhead goes
 up.  If you set the binsize to 0.0, LAMMPS will use the default
 binsize of 1/2 the cutoff.
 
+The *collection/type* option allows you to define custom collections of atom 
+types for the *multi* neighbor mode. By grouping atom types with 
+similar cutoffs, one may be able to improve performance by reducing
+overhead. You must first specify the number of custom collections N to be
+defined followed by N lists of types. Each list consists of a series of type
+ranges separated by commas. The range can be specified as a 
+single numeric value, or a wildcard asterisk can be used to specify a range
+of values.  This takes the form "\*" or "\*n" or "n\*" or "m\*n".  For
+example, if M = the number of atom types, then an asterisk with no numeric 
+values means all types from 1 to M.  A leading asterisk means all types 
+from 1 to n (inclusive).  A trailing asterisk means all types from n to M 
+(inclusive).  A middle asterisk means all types from m to n (inclusive). 
+Note that all atom types must be included in a custom collection.
+
+The *collection/interval* option provides a similar capability. 
+This command allows a user to define custom collections by specifying a 
+series of cutoff intervals. LAMMPS will automatically sort atoms into these intervals
+based on their type-dependent cutoffs or their finite size.
+You must first specify the number of custom collections N to be
+defined followed by N values representing the upper cutoff of each interval.
+This command is particularly useful for granular pairstyles where the interaction distance
+of particles depends on their radius and may not depend on their atom type.
+
```

Comment:
Not clear to me why the user doesn't specify N-1 values to produce N collections.  What happens if some atom is larger than the Nth value and is thus not in any collection?